<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::AppearanceOptionPage</class>
 <widget class="QWidget" name="QtGui::AppearanceOptionPage">
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="showTrafficCheckBox">
     <property name="text">
      <string>Show traffic</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
